connectors/microsoft-teams/src/main/java/io/camunda/connector/model/request/chat/SendMessageInChat.java

```diff
@@ -55,16 +70,28 @@ public boolean equals(final Object o) {
       return false;
     }
     final SendMessageInChat that = (SendMessageInChat) o;
-    return Objects.equals(chatId, that.chatId) && Objects.equals(content, that.content);
+    return Objects.equals(chatId, that.chatId)
+        && Objects.equals(content, that.content)
+        && Objects.equals(bodyType, that.bodyType);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(chatId, content);
+    return Objects.hash(chatId, content, bodyType);
   }
 
   @Override
   public String toString() {
-    return "SendMessageInChat{" + "chatId='" + chatId + "'" + ", content='" + content + "'" + "}";
+    return "SendMessageInChat{"
+        + "chatId='"
+        + chatId
+        + "'"
+        + ", content='"
+        + content
```

Comment:
I think, we should not display `content` and redact messages instead. Otherwise, we might face with GDPR regulations.

Applicable here and other similar places.